.github/workflows/modified_scripts_build.yml

```diff
@@ -58,6 +58,7 @@ jobs:
             python -c 'import sys; print(sys.version)'
           else
             python --version
+            python -m ensurepip
```

Comment:
Pyenv is supposed to install Pip in the course of `pyenv install`. If it doesn't something is wrong.

Reply:
check...  bash function not succsful on pypy 2.7 mac